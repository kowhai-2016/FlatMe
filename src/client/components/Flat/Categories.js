import React from 'react'
import { Link } from 'react-router'

import { Navbar } from 'react-bootstrap'

const Categories = props => {
  const id = props.id
  return (
    <Navbar inverse className='categories-bar'>
      <div><img src='/images/phone.svg' width='25px' /><Link to={`/flat/${id}`}> Contacts</Link></div>
      <div><img src='/images/paper.svg' width='25px' /><Link to={`/flat/${id}/documents`}> Documents</Link></div>
      <div><img src='/images/interface-1.svg' width='25px' /><Link to={`/flat/${id}/notes`}> Notes</Link></div>
    </Navbar>
  )
}

export default Categories
