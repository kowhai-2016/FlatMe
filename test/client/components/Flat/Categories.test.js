/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Categories from '../../../../src/client/components/Flat/Categories'

test('Categories renders correctly', () => {
  const tree = renderer.create(
    <Categories />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
