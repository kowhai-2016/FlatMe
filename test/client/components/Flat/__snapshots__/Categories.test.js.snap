exports[`test Categories renders correctly 1`] = `
<nav
  className="categories-bar navbar navbar-default"
  style={
    Object {
      "margin": "10%",
    }
  }>
  <div
    className="container">
    <div>
      <a
        onClick={[Function]}
        style={Object {}}>
        <img
          src="/images/phone.svg"
          width="25px" />
        <span>
          Contacts
        </span>
      </a>
    </div>
    <div>
      <a
        onClick={[Function]}
        style={Object {}}>
        <img
          src="/images/interface.svg"
          width="25px" />
        <span>
          Documents
        </span>
      </a>
    </div>
    <div>
      <a
        onClick={[Function]}
        style={Object {}}>
        <img
          src="/images/interface-1.svg"
          width="25px" />
        <span>
          Notes
        </span>
      </a>
    </div>
    <div>
      <a
        onClick={[Function]}
        style={Object {}}>
        <img
          src="/images/alarm.svg"
          width="25px" />
        <span>
          Calendar
        </span>
      </a>
    </div>
  </div>
</nav>
`;
